guides/source/i18n.md

```diff
@@ -1139,14 +1139,14 @@ I18n.backend = I18n::Backend::Chain.new(I18n::Backend::ActiveRecord.new, I18n.ba
 
 The I18n API defines the following exceptions that will be raised by backends when the corresponding unexpected conditions occur:
 
-```ruby
-MissingTranslationData       # no translation was found for the requested key
-InvalidLocale                # the locale set to I18n.locale is invalid (e.g. nil)
-InvalidPluralizationData     # a count option was passed but the translation data is not suitable for pluralization
-MissingInterpolationArgument # the translation expects an interpolation argument that has not been passed
-ReservedInterpolationKey     # the translation contains a reserved interpolation variable name (i.e. one of: scope, default)
-UnknownFileType              # the backend does not know how to handle a file type that was added to I18n.load_path
-```
+| Exception  | Reason  |
```

Comment:
I was considering empty headers, it seems they are required, but this is an overall improvement.